src/views/templates/details/tabs/disks/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getTemplateVirtualMachineObject } from '@kubevirt-utils/resources/template';
+
+export const getTemplateVMWithNamespace = (template: V1Template): V1VirtualMachine => {
+  const vm = getTemplateVirtualMachineObject(template);
+
+  if (vm) {
```

Comment:
maybe using produce from `immer` in here would be nicer, wdyt?

Reply:
okay. Done